### PR TITLE
Fix name update and allow for reverse control to be update during edit.

### PR DESCRIPTION
**Pull request by Randy424**
Signed-off-by: randybrunopiverger <rbrunopi@redhat.com>
regarding: https://github.com/stolostron/backlog/issues/21983

- Added logic to prevent change from `reverseExistingRuleName` when editing placement resource label.

### CODE REVIEW

**Comment by Randy424** [comment]
/hold 

_**Still testing**_

**Comment by Randy424** [comment]
We will want to also validate that the issues addressed by this PR are still fixed: https://github.com/stolostron/application-ui/pull/1419

File: frontend/src/routes/Applications/CreateApplication/Subscription/controlData/ControlDataPlacement.js [hunk 1]
```diff
@@ -184,7 +180,8 @@ export const reverseExistingRule = (control, templateObject) => {
 }
 export const reverseExistingRuleName = (control, templateObject) => {
     const active = _.get(templateObject, getSourcePath('Subscription[0].spec.placement.placementRef.name'))
-    if (active && control.type !== 'hidden' && control.active === undefined) {
+
+    if (active && control.active === undefined) {
         const { groupControlData } = control
         const selectedRuleName = groupControlData.find(({ id }) => id === 'selectedRuleName')
         selectedRuleName.active = active.$v
```
Comment (KevinFCormier) [inline frontend/src/routes/Applications/CreateApplication/Subscription/controlData/ControlDataPlacement.js:190]: Can we try eliminating this function and just using `reverseExistingRule` for both the dropdown and the hidden control?

**Comment by Randy424** [comment]
Re-opening PR here: https://github.com/stolostron/console/pull/1636

**Pull request closed without merging**